src/android/InAppBrowser.java

```diff
@@ -781,12 +812,56 @@ private void sendUpdate(JSONObject obj, boolean keepCallback, PluginResult.Statu
         }
     }
 
+    private void addTrustedCA() throws IOException, NoSuchAlgorithmException, CertificateException, KeyManagementException, KeyStoreException {
+        Log.d(LOG_TAG, "Adding trusted certificate if it exists");
+        // Load CAs from an InputStream
+        // (could be from a resource or ByteArrayInputStream or ...)
+        CertificateFactory cf = CertificateFactory.getInstance("X.509");
+        InputStream caInput;
+        try {
+            caInput = this.cordova.getActivity().getAssets().open("www/trusted-ca.der");
+        } catch (IOException ex) {
+            Log.d(LOG_TAG, "No trusted certificate authorities supplied: " + ex.toString());
+            return;
+        }
+        Log.d(LOG_TAG, "Found trusted certificate");
```

Comment:
@grokcode If you run `adb logcat` while running the app, these logs will appear there, prefixed with the value of `LOG_TAG` for grepability. I think `LOG_TAG` is `InAppBrowser` or something similar